channels/us.m3u

```diff
@@ -756,6 +756,8 @@ http://51.195.44.53:8081/tv/nat-geo-wild/playlist.m3u8
 http://161.0.157.5/PLTV/88888888/224/3221227040/index.m3u8
 #EXTINF:-1 tvg-id="NBC.us" tvg-name="NBC" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Local",NBC (720p)
 http://161.0.157.8/PLTV/88888888/224/3221227040/index.m3u8
+#EXTINF:-1 tvg-id="NBC8.us" tvg-name="NBC 8 (WGAL)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/7b8oEUJ.png" group-title="Local",NBC 8 (WGAL)
+https://cdnapisec.kaltura.com/p/2630451/sp/263045100/playManifest/entryId/1_qued8u74/protocol/https/format/applehttp/flavorIds/1_cy0ldm01,1_swt14n9f,1_18ckz32v,1_5wgmiaam,1_wz17gs30,1_5t8hyxic,1_r6w5x7ae/ks/djJ8MjYzMDQ1MXz4HxGcfqR6k6BQXNBx_8X2-hKbSgrC8pnWsVccUrW9Sw7b0pi4ItF_LrRUaHjKTeDWIvp547cXqztYdpSwUqHDUEdIMnvufB11DCTpQVpJgIoifpOUsDHkcvCK4MxaRgA0VQpdRLvEMSEFlJ-nAzvoBWztaAs3_Xxm9EO9FX2pXA==/a.m3u8
```

Comment:
The link doesn't work.

Reply:
Sorry, I pasted the wrong link. Here You Go: https://cdnapisec.kaltura.com/p/2630451/sp/263045100/playManifest/entryId/1_n55lj49n/format/applehttp/protocol/https/uiConfId/47253602/a.m3u8

Reply:
It takes a couple of seconds for the audio to be clear.